src/foam/mlang/mlang.js

```diff
@@ -441,29 +441,30 @@ foam.CLASS({
     },
     {
       name: 'authorize',
-      javaCode:`
-  return true;
-  `
+      javaCode: `//noop`
     },
     {
-      name:'authorizeArg',
-      javaCode:`
+      name: 'authorizeArg',
+      javaCode: `
   if ( arg instanceof foam.core.PropertyInfo ) {
```

Comment:
I don't think this check is needed. Or this method. Just move this code to AbstractPropertyInfo and then rely on polymorphism to handle this for you. Then instead of calling authorizeArg just call arg.authorize().

Reply:
Resolved by 0ec5a4592.